noxfile.py

```diff
@@ -160,3 +160,27 @@ def mypy(session):
     session.install("mypy >= 1.10.0", "pip")
     session.install("-r", "requirements.txt")
     session.run(*mypy_command, *mypy_options, *session.posargs)
+
+
+@nox.session(name="import")
+def try_import(session):
```

Comment:
📝 `import` is a reserved word in Python, but nox lets us define a name for a session which is different from the function name.  Pretty helpful!